src/cpuid/src/amd/normalize.rs

```diff
@@ -116,7 +130,20 @@ impl super::AmdCpuid {
     }
 
     /// Process CPUID.
```

Comment:
Should we leave this as it is even after changing the function name to `passthrough_cache_topology()`?

Reply:
Updated comment to:
```rust
    /// Passthrough cache topology.
```